UnityProject/Assets/Scripts/Messages/Client/NewPlayer/DoorNewPlayer.cs

```diff
@@ -7,12 +7,14 @@ public class DoorNewPlayer: ClientMessage
 
 	public override void Process()
 	{
-		LoadNetworkObject(Door);
-		var doorController = NetworkObject.GetComponent<DoorController>();
-
-		if (doorController != null)
+		// LoadNetworkObject returns bool, so it can be used to check if object is loaded correctly
+		if (LoadNetworkObject(Door))
 		{
-			doorController.UpdateNewPlayer(SentByPlayer.Connection);
+			// https://docs.unity3d.com/2019.3/Documentation/ScriptReference/Component.TryGetComponent.html
+			if (NetworkObject.TryGetComponent(out DoorController doorController))
+			{
+				doorController.UpdateNewPlayer(SentByPlayer.Connection);
+			}
 		}
```

Comment:
NullReferenceException: Object reference not set to an instance of an object
DoorNewPlayer.Process () (at /github/workspace/UnityProject/Assets/Scripts/Messages/Client/NewPlayer/DoorNewPlayer.cs:11)
GameMessageBase.Process (Mirror.NetworkConnection sentBy) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:25)
ClientMessage.Process (Mirror.NetworkConnection sentBy) (at /github/workspace/UnityProject/Assets/Scripts/Messages/Client/ClientMessage.cs:14)
GameMessageBase.PreProcess (Mirror.NetworkConnection sentBy, GameMessageBase b) (at /github/workspace/UnityProject/Assets/Scripts/Messages/GameMessageBase.cs:18)
Mirror.MessagePacker+<>c__DisplayClass6_0`2[T,C].<MessageHandler>b__0 (Mirror.NetworkConnection conn, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/MessagePacker.cs:146)
Mirror.NetworkConnection.InvokeHandler (System.Int32 msgType, Mirror.NetworkReader reader, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:219)
Mirror.NetworkConnection.TransportReceive (System.ArraySegment`1[T] buffer, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkConnection.cs:272)
Mirror.NetworkServer.OnDataReceived (System.Int32 connectionId, System.ArraySegment`1[T] data, System.Int32 channelId) (at /github/workspace/UnityProject/Assets/Mirror/Runtime/NetworkServer.cs:511)
UnityEngine.Events.InvokableCall`3[T1,T2,T3].Invoke (T1 args0, T2 args1, T3 args2) (at /home/builduser/buildslave/unity/build/Runtime/Export/UnityEvent/UnityEvent.cs:284)
UnityEngine.Events.UnityEvent`3[T0,T1,T2].Invoke (T0 arg0, T1 arg1, T2 arg2) (at /home/builduser/buildslave/unity/build/Runtime/Export/UnityEvent/UnityEvent/UnityEvent_3.cs:58)
Mirror.TelepathyTransport.ProcessServerMessage () (at /github/workspace/UnityProject/Assets/Mirror/Runtime/Transport/TelepathyTransport.cs:159)
M